ibm/provider/provider.go

```diff
@@ -1067,6 +1067,7 @@ func Provider() *schema.Provider {
 			// bare_metal_server
 			"ibm_is_bare_metal_server_action":                        vpc.ResourceIBMIsBareMetalServerAction(),
 			"ibm_is_bare_metal_server_disk":                          vpc.ResourceIBMIsBareMetalServerDisk(),
+			"ibm_is_bare_metal_server_reload":                        vpc.ResourceIBMIsBareMetalServerReload(),
 			"ibm_is_bare_metal_server_network_attachment":            vpc.ResourceIBMIsBareMetalServerNetworkAttachment(),
```

Comment:
resource name should be `ibm_is_bare_metal_server_initialization` as per discussion and put in SRB https://github.ibm.com/riethm/srb/blob/70f1717533158dded42572a83ff83dd75b97b923/proposals/3153/README.md#3123-terraform-and-packer